source/MaterialXRuntime/Tokens.h

```diff
@@ -0,0 +1,58 @@
+//
+// TM & (c) 2019 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_TOKENS_H
+#define MATERIALX_TOKENS_H
+
+/// @file
+/// TODO: Docs
+
+#include <MaterialXRuntime/RtToken.h>
+
+namespace MaterialX
+{
+
+/// Namespace of commonly used tokens.
+namespace Tokens
```

Comment:
Namespace collecting all commonly used tokens in one place.

Reply:
Seems useful!